app/models/admin_blacklisted_email.rb

```diff
@@ -1,7 +1,7 @@
 class AdminBlacklistedEmail < ApplicationRecord
   before_validation :canonicalize_email
 
-  validates :email, presence: true, uniqueness: true, email_veracity: true
+  validates :email, presence: true, uniqueness: true, email_format: { case_sensitive: true }
```

Comment:
`case_sensitive` is not an option in EmailFormatValidator (only `options[:allow_blank]` is checked there). Just `true` should be fine.

Reply:
Thanks for pointing out the problem. That was my failed attempt at addressing this deprecation warning:
```   DEPRECATION WARNING: Uniqueness validator will no longer enforce case sensitive comparison in Rails 6.1. To continue case sensitive comparison on the :email attribute in AdminBlacklistedEmail model, pass `case_sensitive: true` option explicitly to the uniqueness validator. (called from validate_each at /usr/local/bundle/bundler/gems/globalize-76b808b13d15/lib/patches/active_record/rails5_1/uniqueness_validator.rb:38)```

However, I mistakenly added `case_sensitive` to `email_format` where I should added it to `uniqueness`. So it seems like a backward compatible validation (assuming it is supposed to be case-sensitive) would be:

`  validates :email, presence: true, uniqueness: { case_sensitive: true }, email_format: true`

Reply:
Let's use `case_sensitive: false` for the uniqueness check. That's our usual approach for emails, for example:

https://github.com/otwcode/otwarchive/blob/e7c7da0a33a554cfde124ef1ad2e3df5ce005470/app/models/invite_request.rb#L5

https://github.com/otwcode/otwarchive/blob/e7c7da0a33a554cfde124ef1ad2e3df5ce005470/app/models/user.rb#L201

Reply:
Sure. It's been done in commits updated today.